bagitobjecttransfer/recordtransfer/templates/recordtransfer/email/account_updated.html

```diff
@@ -15,9 +16,9 @@
         <div style="
             padding: 0px 20px;">
             <div>
-                <p>Your {{ changed_item }} for the NCTR Record Transfer Portal has been {{ changed_status }}.</p>
+                <p>{% blocktranslate %}Your {{ changed_item }} for the NCTR Record Transfer Portal has been {{ changed_status }}.{% endblocktranslate %}</p>
```

Comment:
Thanks for adding this, no changes needed here I just wanted to mention that I want to do this for all the other emails at some point, too.